examples/libjson-rpc-cpp/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Anyway, feel free to delete this PR and do it your way. I just wanted to start some tests.